service/history/transferQueueTaskExecutorBase.go

```diff
@@ -49,7 +49,7 @@ import (
 )
 
 const (
-	transferActiveTaskDefaultTimeout = 3 * time.Second
+	transferActiveTaskDefaultTimeout = 60 * time.Second
```

Comment:
60s is too long can we try 10s

Reply:
should this actually be adaptive?  Low when we're running well, but high(er) when there's a backlog to prevent the "stampede" (love that word!).

Reply:
valid point, but would increase complexity. definitely worth consider in the future.